AUTHORS

```diff
@@ -3139,5 +3139,188 @@ Shigma <33423008+Shigma@users.noreply.github.com>
+Anders Kaseorg <andersk@mit.edu>
+Hollow Man <hollowman@hollowman.ml>
+nlf <quitlahok@gmail.com>
+naortedgi <naor.tedgi@gmail.com>
+Narasimha Prasanna HN <yogaimpression@gmail.com>
+Zijian Liu <Lxxyxzj@gmail.com>
+inokawa <48897392+inokawa@users.noreply.github.com>
+Michael Bashurov <accounts@saitonakamura.me>
+Moshe vilner <shootermv@gmail.com>
+Nicolai Stange <nicstange@gmail.com>
+kai zhu <kaizhu256@gmail.com>
+FrankQiu <iam.frankqiu@gmail.com>
+Rock <arockplaysmc@gmail.com>
+Chinmoy Chakraborty <chinmoy12c@gmail.com>
+Maksym Baranovskyi <mkbaranovskyi@gmail.com>
+Michael Chen <4326639+mcgitty@users.noreply.github.com>
+François-Denis Gonthier <francois-denis.gonthier@opersys.com>
+Dr <dr@dr.run>
+Nitzan Uziely <nitzan@testim.io>
+Adrien Maret <adrien@maret.email>
+Thiago Padilha <thiago@padilha.cc>
+Joseph Hackman <josephhackman@gmail.com>
+Pranshu Jethmalani <pmj642@gmail.com>
+Rohan Chougule <rohan183chougule@gmail.com>
+Mohamed Kamagate <kamagatos@gmail.com>
+Ajay Poshak <poshakajay@gmail.com>
+Isaac Levy <isaac.r.levy@gmail.com>
+ugultopu <ugultopu@users.noreply.github.com>
+Nicholas Schamberg <schamberg.nicholas@gmail.com>
+Dimitris Halatsis <dimitris_ha@hotmail.com>
+Mattia Pontonio <44380480+mattiapontonio@users.noreply.github.com>
+Milad Fa <mfarazma@redhat.com>
+Emil Sivervik <emil@sivervik.com>
+alexbs <alexbs@users.noreply.github.com>
+Ian Storm Taylor <ian@ianstormtaylor.com>
+Carlos Fuentes <carlos.fuentes.rios.95@gmail.com>
+Tyler Ang-Wanek <tylerw@axosoft.com>
+Matthew Mario Di Pasquale <liveloveprosper@gmail.com>
+ttzztztz <yangziyue80@outlook.com>
+Romuald Brillout <github@brillout.com>
+Dave Cardwell <hello@davecardwell.com>
+Akash Negi <55234838+NegiAkash890@users.noreply.github.com>
+James Addison <jay@jp-hosting.net>
+linkgoron <linkgoron@gmail.com>
+Fabian Cook <hello@fabiancook.dev>
+Kalvin Vasconcellos <kalvin.vasconcellos@gmail.com>
+marsonya <akhil.marsonya27@gmail.com>
+Qingyu Deng <bitdqy@hotmail.com>
+Matin Zadehdolatabad <zadehdolatabad@gmail.com>
+Daniel Clark <daniec@microsoft.com>
+Sajal Khandelwal <skhandelwa22@bloomberg.net>
+Cheng Liu <liucheng.tech@outlook.com>
+Utku Gultopu <ugultopu@gmail.com>
+Jay Tailor <60511316+JayvaScript@users.noreply.github.com>
+Greg Ziskind <gregory.ziskind@synergistcomputing.com>
+Dan Čermák <dcermak@suse.com>
+ttzztztz <ttzztztz@outlook.com>
+Vít Ondruch <vondruch@redhat.com>
+humanwebpl <58517331+humanwebpl@users.noreply.github.com>
+Dawid Rusnak <dawid@drcode.pl>
+obi-el <elobiobinna@gmail.com>
+Merlin Luntke <22600241+Luntke@users.noreply.github.com>
+Marko Kaznovac <kaznovac@users.noreply.github.com>
+Gabriel Schulhof <gabrielschulhof@gmail.com>
+Ian Kerins <ianskerins@gmail.com>
+dbachko <dbachko@gmail.com>
+Mattias Buelens <mattias@buelens.com>
+Dylan Elliott <dylanelliott27@hotmail.com>
+Wassim Chegham <github@wassim.dev>
+simov <simeonvelichkov@gmail.com>
+wwwzbwcom <zbwhome@outlook.com>
+Stephen Belanger <stephen.belanger@datadoghq.com>
```

Comment:
@Qard  Looks like you've used two email addresses so I imagine we'll want to put one in .mailmap. Do you care which one is listed in AUTHORS?

Reply:
I went with preserving the previous one and removing this one, but it's a simple .mailmap change if @qard wants it the other way around.